infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -95,8 +93,9 @@
      */
     public ManagementServer(ServerContext serverContext) {
 
-        this.opts = serverContext.getServerConfig();
-        this.localEndpoint = this.opts.get("--address") + ":" + this.opts.get("<port>");
+        this.managementServerConfig = ManagementServerConfig.parse(serverContext.getServerConfig());
+
+        this.localEndpoint = NodeLocator.getLegacyEndpoint(managementServerConfig.getNodeLocator());
```

Comment:
Looks like localEndpoint is not used at all, if I am right please get rid of it

Reply:
done